src/main/java/training/subsystems/elbow/ElbowSimulation.java

```diff
@@ -0,0 +1,55 @@
+package training.subsystems.elbow;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.PositionVoltage;
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+import utils.simulation.SingleJointedArmSimulation;
+
+public class ElbowSimulation implements IElbow {
+
+	private SingleJointedArmSimulation elbowSimulation;
+	private ArmFeedforward armFeedforward;
+	private PIDController controller;
+	private PositionVoltage positionVoltage;
+
+	public ElbowSimulation() {
+		SingleJointedArmSim armSim = new SingleJointedArmSim(
+			DCMotor.getFalcon500(SimulationElbowConstants.NUMBER_OF_MOTORS),
```

Comment:
shouldn't this be getNeo() or smthing?

Reply:
dror- nope, this constructor was copied from cressendo